src/main/java/com/couchbase/lite/Attachment.java

```diff
@@ -202,7 +202,7 @@ protected InputStream getBodyIfNew() {
                     BlobStoreWriter writer;
                     try {
                         writer = blobStoreWriterForBody(body, database);
-                    } catch (IOException e) {
+                    } catch (Exception e) {
                         throw new CouchbaseLiteException(e.getMessage(), Status.ATTACHMENT_ERROR);
```

Comment:
I would think this is enough. iOS doesn't have anything special for this as well.